server/controllers/reports_proposed/data/account_result.js

```diff
@@ -0,0 +1,239 @@
+// reports_proposed/data/bilan.js
+// Collects and aggregates data for the enterprise bilan
+
+var q       = require('q');
+var db      = require('../../../lib/db');
+var numeral = require('numeral');
+
+var formatDollar = '$0,0.00';
+var resultAccountDate = new Date();
+
+// expose the http route
+exports.compile = function (options) {
+  'use strict';
+
+  var deferred = q.defer(), context = {}, infos = {}, chargeData = {}, profitData = {};
+  context.reportDate = resultAccountDate.toDateString();
+
+  var sql =
+    'SELECT `acc`.`id` AS `accountId`, `acc`.`account_txt` AS `accounTxt`, `acc`.`account_number` AS `accountNumber`, ' +
```

Comment:
It would be best not to use the "`" tick marks anymore.  It makes the code difficult to read.